index.html

```diff
@@ -4,7 +4,82 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+      integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css?family=Rock+Salt"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link href="asset/reset.css" rel="stylesheet" type="text/css" />
+    <link href="asset/style.css" rel="stylesheet" type="text/css" />
     <title>Document</title>
   </head>
-  <body></body>
+  <body>
+    <header>
+      <nav>
+        <ul class="navigation">
+          <li class="item nav_item_1"><a href="#">logo</a></li>
+          <li class="item nav_item_2"><a href="#">Sign up</a></li>
+          <li class="item nav_item_3"><a href="#">About</a></li>
+          <li class="item nav_item_4"><a href="#">Contact</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="wrapper container">
+        <div class="form-container">
+          <form id="myForm">
+            <label>Title:</label><br />
+            <input
+              class="field"
+              type="text"
+              name="title"
+              value=""
+              placeholder="Enter the title"
+            /><br />
+            <label>Author:</label><br />
+            <input
+              class="field"
+              type="text"
+              name="author"
+              value=""
+              placeholder="Who's the author"
+            /><br />
+            <label>Number of pages: </label><br />
+            <input
+              class="field"
+              type="number"
+              name="pages"
+              placeholder="No of pages"
+            /><br />
```

Comment:
Hello great job for your project, please look for a way to control the submission of an empty form. "Empty" books can be added. You either add the option `required` to these inputs, or you control that with javascript